pkgs/applications/networking/cluster/helmfile/default.nix

```diff
@@ -19,9 +25,14 @@ buildGoModule rec {
 
   ldflags = [ "-s" "-w" "-X go.szostok.io/version.version=v${version}" ];
 
-  nativeBuildInputs = [ installShellFiles ];
+  nativeBuildInputs =
+    [ installShellFiles ] ++
+    lib.optionalString (pluginsDir != null) [ makeWrapper ];
```

Comment:
I don't think `optionalString` will work here, you might need to use the full `if ... then ... else` syntax

Reply:
Yep... I meant `lib.optionals`. Thanks.